comments-bundle/src/Resources/contao/classes/Comments.php

```diff
@@ -293,12 +293,6 @@ protected function renderCommentForm(FrontendTemplate $objTemplate, \stdClass $o
 
 			if ($flashBag->has('comment_added'))
 			{
-				/** @var PageModel $objPage */
-				global $objPage;
-
-				$objPage->noSearch = 1;
```

Comment:
No need to set `noSearch` here. The template is except from the indexer anyway and disabling the cache is not needed as the session is started.

Reply:
All the other appearances will be removed with https://github.com/contao/contao/pull/3085.